wp-content/plugins/core/src/Templates/Controllers/Block/Hero.php

```diff
@@ -0,0 +1,35 @@
+<?php
+declare( strict_types=1 );
+
+namespace Tribe\Project\Templates\Controllers\Block;
+
+class Hero extends Block_Controller {
+	public function render( string $path = '' ): string {
```

Comment:
nuke now and actually get the twig file going here?

Reply:
Yep, will do once merged and I start working on this